core/drivers/crypto/caam/cipher/caam_cipher.c

```diff
@@ -973,9 +944,9 @@ static TEE_Result do_update_cipher(struct drvcrypt_cipher_update *dupdate)
 
 		dstbuf.length = dupdate->dst.length - offset;
 
-		retstatus = caam_cipher_block(ctx, true, NEED_KEY1,
-					      ctx->encrypt, &srcbuf, &dstbuf,
-					      false);
+		retstatus =
```

Comment:
Strange line break.

Reply:
I agree, this looks weird. I might tune ```PenaltyBreakAssignment``` parameter of our ```.clang-format``` configuration file for the next time.